CHANGELOG.md

```diff
@@ -2,28 +2,38 @@
 * [FEATURE] Global limit on the max series per user and metric #1760
   * `-ingester.max-global-series-per-user`
   * `-ingester.max-global-series-per-metric`
   * Requires `-distributor.replication-factor` and `-distributor.shard-by-all-labels` set for the ingesters too
 * [FEATURE] Flush chunks with stale markers early with `ingester.max-stale-chunk-idle`. #1759
 * [FEATURE] EXPERIMENTAL: Added new KV Store backend based on memberlist library. Components can gossip about tokens and ingester states, instead of using Consul or Etcd. #1721
+* [FEATURE] EXPERIMENTAL: Use TSDB in the ingesters & flush blocks to S3/GCS ala Thanos. This will let us use an Object Store more efficiently and reduce costs. #1695
 * [FEATURE] Allow Query Frontend to log slow queries with `frontend.log-queries-longer-than`. #1744
-* [FEATURE] The frontend split and cache intervals can now be configured using the respective flag `--querier.split-queries-by-interval` and `--frontend.cache-split-interval`.
-  * If `--querier.split-queries-by-interval` is not provided request splitting is disabled by default.
-  * __`--querier.split-queries-by-day` is still accepted for backward compatibility but has been deprecated. You should now use `--querier.split-queries-by-interval`. We recommend a to use a multiple of 24 hours.__
-* [ENHANCEMENT] Allocation improvements in adding samples to Chunk. #1706
+* [FEATURE] Add HTTP handler to trigger ingester flush & shutdown - used when running as a stateful set with the WAL enabled.  #1746
+* [ENHANCEMENT] Reduce memory allocations in the write path. #1706
 * [ENHANCEMENT] Consul client now follows recommended practices for blocking queries wrt returned Index value. #1708
 * [ENHANCEMENT] Consul client can optionally rate-limit itself during Watch (used e.g. by ring watchers) and WatchPrefix (used by HA feature) operations. Rate limiting is disabled by default. New flags added: `--consul.watch-rate-limit`, and `--consul.watch-burst-size`. #1708
 * [ENHANCEMENT] Added jitter to HA deduping heartbeats, configure using `distributor.ha-tracker.update-timeout-jitter-max` #1534
-* [ENHANCEMENT] Allocation improvements in adding samples to Chunk. #1706
-* [ENHANCEMENT] Consul client now follows recommended practices for blocking queries wrt returned Index value. #1708
-* [ENHANCEMENT] Consul client can optionally rate-limit itself during Watch (used e.g. by ring watchers) and WatchPrefix (used by HA feature) operations. Rate limiting is disabled by default. New flags added: `--consul.watch-rate-limit`, and `--consul.watch-burst-size`. #1708
+* [ENHANCEMENT] Add ability to flush chunks with stale markers early. #1759
+* [BUGFIX] Stop reporting successful actions as 500 errors in KV store metrics. #1798
+* [BUGFIX] Fix bug where duplicate labels can be returned through metadata APIs. #1790
+* [BUGFIX] Fix reading of old, v3 chunk data. #1779
+
+In this release we updated the following dependencies:
+- gRPC v1.25.0
```

Comment:
We could point out this can significantly reduce ingester CPU usage when compression is turned on.
(I got a 30% drop in prod)